index.html

```diff
@@ -184,4 +184,11 @@ <h2>What do participants think about our Mentorship Programme?</h2>
             </a>
           </div>
     </div>
+
+    <div class="support">
+        <h2>Experiencing technical issues?</h2>
+        <div class="text-center">
+            <a href="https://docs.google.com/forms/d/e/1FAIpQLScf4lfJbRVhZHADmcOFR3PoXstlj1dmTzdUT7VAo7WjMHj9cA/viewform" target="_blank" class="btn btn-primary">Send us a report</a>
```

Comment:
Is that possible to redirect user to the form to report a bug in github repo (so the person can report issue directly in repo)?

Reply:
yes, actually it can be done like this: 

```suggestion
            <a href="https://github.com/WomenWhoCode/london/issues/new/choose" target="_blank" class="btn btn-primary">Send us a report</a>
```

Reply:
another option is to redirect to create bug: `https://github.com/WomenWhoCode/london/issues/new?template=bug_report.md&title=bug title`